crates/tx5-connection/src/webrtc/libdatachannel.rs

```diff
@@ -25,27 +25,29 @@ struct Dch(CloseSend<Cmd>);
 
 impl datachannel::DataChannelHandler for Dch {
     fn on_open(&mut self) {
-        self.0.send_or_close(Cmd::DataChanOpen);
+        let _ = self.0.send_or_close(Cmd::DataChanOpen);
     }
 
     fn on_closed(&mut self) {
-        self.0
+        let _ = self
+            .0
             .send_or_close(Cmd::Error(std::io::Error::other("DataChanClosed")));
     }
 
     fn on_error(&mut self, err: &str) {
-        self.0
-            .send_or_close(Cmd::Error(std::io::Error::other(format!(
-                "DataChanError: {err}"
-            ))));
+        let _ =
+            self.0
+                .send_or_close(Cmd::Error(std::io::Error::other(format!(
+                    "DataChanError: {err}"
+                ))));
     }
 
     fn on_message(&mut self, msg: &[u8]) {
-        self.0.send_or_close(Cmd::RecvMessage(msg.to_vec()));
+        let _ = self.0.send_or_close(Cmd::RecvMessage(msg.to_vec()));
```

Comment:
Worth logging the error here?

Reply:
I waffled back and forth on both of these, but ultimately, the only error we get is that the channel was closed, which might be because of a filled channel buffer, but equally could just be because the app requested the connection close. All events emitted by the underlying webrtc backend would emit errors that confuse debuggers. I'm going to leave this not emitting for now, but we can always retool this flow if we run into something hard to debug.